src/libnnstreamer-edge/nnstreamer-edge-common.c

```diff
@@ -368,7 +367,7 @@ nns_edge_data_copy (nns_edge_data_h data_h, nns_edge_data_h * new_data_h)
     copied->data[i].data = nns_edge_memdup (ed->data[i].data,
         ed->data[i].data_len);
     copied->data[i].data_len = ed->data[i].data_len;
-    copied->data[i].destroy_cb = g_free;
+    copied->data[i].destroy_cb = free;
```

Comment:
Then, let's add a null check when destroy_cb is called.
Or, write a static function that does a null check here.

Reply:
add new function for this.